gulpfile.babel.js

```diff
@@ -20,6 +20,7 @@ import fs from 'fs';
 import punycode from 'punycode';
 
 import del from 'del';
```

Comment:
shouldn't this import be removed? why is linter not complaining?

Reply:
Yeah the linter didn't catch it :( Thanks Arnau - import removed now.